webapp/src/components/library/component/LibraryTable.tsx

```diff
@@ -0,0 +1,84 @@
+import {Col, Container, Row} from "react-bootstrap";
+import React from "react";
+import { History } from 'history';
+import moment from 'moment';
+
+import "./LibraryTable.css";
+import {useHistory} from "react-router";
+
+export interface LickInterface {
+    id: number
+    name: string
+    description: string
+    dateUploaded: string
+    audioFileLocation: string
+    audioLength: number
+    tab: string,
+    tuning: string
+    isPublic: boolean,
+    owner: any
+}
+
+export interface LibraryTableProps {
+    licks: LickInterface[]
+    selected?: LickInterface
+    setSelected: Function
+}
+
+export default function LibraryTable(props: LibraryTableProps) {
+    const selected = props.selected;
+    const setSelected = props.setSelected;
+
+    const history = useHistory();
+    if (!props.licks || props.licks.length < 1) {
+        return (<div>We could not find any licks associated with your account.</div>)
+    }
+
+    return (
+        <Container fluid className="library-table">
+            <Row className="table-header">
+                <Col xs={5}>Name</Col>
+                <Col xs={2}>Length</Col>
+                <Col xs={3}>Created Date</Col>
+                <Col xs={2}>Actions</Col>
+            </Row>
+            <div className="scrollable">
+                {renderTableBody(props.licks, selected, setSelected, history)}
+            </div>
+        </Container>
+    )
+}
+
+function renderTableBody(data: LickInterface[],
+                         selected: LickInterface | undefined,
+                         setSelected: Function,
+                         history: History) {
+
+    if (data) {
```

Comment:
do you need to check for data here if you already check if props.licks and props.licks.length >=1?